docs/mindsdb-docs/docs/sql/project.md

```diff
@@ -0,0 +1,372 @@
+# The `PROJECT` Entity
+
+MindsDB introduces the `PROJECT` entity that lets you create projects to store your models.
```

Comment:
Since the project will store more things then just `models` maybe we can say something like `to store your ML experiments`?

Reply:
Ok.